modules/oc-mirror-workflows-partially-disconnected-v2.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-partially-disconnected-v2_{context}"]
+= Mirroring an image set in a partially disconnected environment
+
+.Mirror To Mirror
+
+You can use this workflow for partially disconnected mirroring, that is, when the environment from which you are executing oc-mirror has both:
+- Access to the internet
+- Access to the mirror registry
+
+.Procedure
+
+. Copy the container images from the source specified in the image set configuration to the destination, that is, container registry.
++
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --workspace file://oc-mirror/mirror1 docker://localhost:6000 --v2
+----
```

Comment:
@sherine-k do we need to mention --dest-tls-verify=false  or is that not required ?